geometry_msgs/msg/PolygonInstanceStamped.msg

```diff
@@ -0,0 +1,5 @@
+# This represents a Polygon with reference coordinate frame and timestamp
+# Includes and unique identification field for disambiguating multiple instances
+
+geometry_msgs/PolygonStamped polygon
```

Comment:
Not sure how this works in ROS 2, but in ROS 1 there is an explicit requirement to have header as the first field to qualify for the special handling of stamped messages.

Reply:
I believe it is using the same logic in ROS 2, I would expect:

```
std_msgs/Header header
geometry_msgs/PolygonInstance polygon
```

Reply:
Agreed, change made